context/app/static/js/components/detailPage/provenance/ProvVis/ProvVis.jsx

```diff
@@ -15,7 +15,7 @@ export default function ProvVis({ provData, getNameForActivity, getNameForEntity
   const { steps, setSteps } = useProvenanceStore(useProvenanceStoreSelector);
 
   useEffect(() => {
-    setSteps(new ProvData(provData, getNameForActivity, getNameForEntity, entity_type).toCwl());
+    setSteps(new ProvData(provData, entity_type, getNameForActivity, getNameForEntity).toCwl());
```

Comment:
What prompted the argument reordering?

Reply:
https://eslint.org/docs/latest/rules/default-param-last

Since `ProvData` has default arguments defined for `getNameForActivity` and `getNameForEntity`, the convention is to put them last so they can be omitted entirely; otherwise, there's no way to define `entity_type` without also providing values for the other two arguments.

Reply:
Alternatively - we could change the constructor of `ProvData` to take in an object with these args instead of having them be positional (i.e. so the constructor is `new ProvData({provData, entity_type, getNameForActivity, getNameForEntity})`, we can have any of them be optional without worrying about their order; we'd still need to modify all the calls to `new ProvData` accordingly, but this idea seems a bit more maintainable in the long run.

Reply:
Preference to use an object, but shouldn't block here.

Reply:
I've made this adjustment, I prefer that alternative approach as well. The only downside is needing to make sure that the `prov` prop is actually being passed in as `prov` (which I've adjusted accordingly where needed)